priv/repo/migrations/20190519175341_add-name-to-groups.exs

```diff
@@ -0,0 +1,9 @@
+defmodule :"Elixir.FeedbackApi.Repo.Migrations.Add-name-to-groups" do
```

Comment:
Looks like it doesn't love dashes in migration names.

Reply:
Yeah, I'll be more cognizant of that going forward, sorry.